config/config.go

```diff
@@ -45,19 +45,63 @@ func parseFile(c *Config, content []byte) error {
 	return nil
 }
 
-func isValidCheck(check string, checks []string) bool {
-	for _, validCheck := range checks {
-		if strings.EqualFold(check, validCheck) {
-			return true
-		}
-	}
-	return false
+type CheckName = string
+
+// Redefining check names here to avoid circular imports.
+const (
+	BinaryArtifacts      CheckName = "Binary-Artifacts"
+	BranchProtection     CheckName = "Branch-Protection"
+	CIIBestPractices     CheckName = "CII-Best-Practices"
+	CITests              CheckName = "CI-Tests"
+	CodeReview           CheckName = "Code-Review"
+	Contributors         CheckName = "Contributors"
+	DangerousWorkflow    CheckName = "Dangerous-Workflow"
+	DependencyUpdateTool CheckName = "Dependency-Update-Tool"
+	Fuzzing              CheckName = "Fuzzing"
+	License              CheckName = "License"
+	Maintained           CheckName = "Maintained"
+	Packaging            CheckName = "Packaging"
+	PinnedDependencies   CheckName = "Pinned-Dependencies"
+	SAST                 CheckName = "SAST"
+	SBOM                 CheckName = "SBOM"
+	SecurityPolicy       CheckName = "Security-Policy"
+	SignedReleases       CheckName = "Signed-Releases"
+	TokenPermissions     CheckName = "Token-Permissions"
+	Vulnerabilities      CheckName = "Vulnerabilities"
+	Webhooks             CheckName = "Webhooks"
+)
```

Comment:
Is there any way we can de-duplicate from 

https://github.com/ossf/scorecard/blob/20ec42c2b5b993e66293e6b34c04cf554e4990c8/internal/probes/probes.go#L25-L50

maybe by moving to `internal/checknames/checknames.go` or something? and referencing those here?

If redefined here, they shouldn't be public.

Reply:
I made them private, but I think creating a new module will clutter up `utils` and lead to a lot more import statements.

If the issue with duplication is that a new check writer will have to update several check lists, we could have an E2E test that imports all these modules and checks they have the same lists? But that would require them to be public.

Reply:
> clutter up utils

I'm not sure which `utils` package you're talking about.

> lead to a lot more import statements.

it's being used in 2 spots, I dont think having 1 new import in 2 files is a problem. and seems simpler than the e2e test you're considering adding.